xwiki-platform-core/xwiki-platform-attachment/xwiki-platform-attachment-validation/xwiki-platform-attachment-validation-api/src/main/java/org/xwiki/attachment/validation/AttachmentValidationConfiguration.java

```diff
@@ -71,4 +71,10 @@ public interface AttachmentValidationConfiguration
      * @since 14.10.13
      */
     long getMaxUploadSize(EntityReference entityReference);
+
+    /**
+     * @return whether or not summaries for attachments should be settable and displayed from UI
+     * @since 16.3.0RC1
+     */
+    boolean areSummariesEnabled();
```

Comment:
Why are summaries config related to attachment validation? What could be related to validation is if we have a config saying that a summary is mandatory for accepting a new attachement but this is not the case here AFAICS.

Reply:
Creating a completely new script service just for that setting felt wrong somehow (so I went with `AttachmentValidationScriptService`), but I agree that the end result doesn't make a lot of sense semantically.
I guess a better option would be to find a more fitting configuration service to extend, but I'm not sure which one.

Reply:
It's fine to add missing classes (there was no need before). So right now we need a new configuration class for attachments AFAICS and a way to return that config from a new script service (might be better to return the full config class than duplicate the config methods, unless some methods are supposed to be not visible from a SS).

Reply:
+1 for moving that elsewhere